extra-utils/starship/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME=starship
+PKGSEC=utils
+PKGDES="A customizable prompt for shell."
+PKGDEP="glibc openssl libgit2"
+PKGSUG="bash zsh fish powershell tcsh elvish"
+BUILDDEP="rustc llvm python-3"
+ABTYPE=rust
+USECLANG=1
+ABSPLITDBG=0
```

Comment:
```suggestion

ABTYPE=rust
USECLANG=1
ABSPLITDBG=0
```

Usually we use a blank line to separate metadata definitions and compilation definitions.